---
version: 1
interactions:
- request:
    body: <CreateBucketConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><LocationConstraint>fr-par</LocationConstraint></CreateBucketConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;host;x-amz-acl;x-amz-content-sha256;x-amz-date,
        Signature=8479ea7d28d62a09413e8fcde33483034747609c2c394e28130d2af8a924f5ff
      Content-Length:
      - "150"
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Acl:
      - private
      X-Amz-Content-Sha256:
      - 2cb57fad7b7168921a4c94426cfcb9ee2953f126430595df844e22d50f029060
      X-Amz-Date:
      - 20211108T150913Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/
    method: PUT
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Mon, 08 Nov 2021 15:09:16 GMT
      Location:
      - /test-acc-scaleway-object-cors-empty-origin-8035963149870005357
      X-Amz-Id-2:
      - tx8c7edfcfcd1c4f5387c01-0061893d9b
      X-Amz-Request-Id:
      - tx8c7edfcfcd1c4f5387c01-0061893d9b
    status: 200 OK
    code: 200
    duration: ""
- request:
    body: ""
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-md5;host;x-amz-acl;x-amz-content-sha256;x-amz-date,
        Signature=3f56db55f7a04203ffe099c8d7afd54fddfaf08002e902fe171ad5b29854c46f
      Content-Md5:
      - 1B2M2Y8AsgTpgAmY7PhCfg==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Acl:
      - private
      X-Amz-Content-Sha256:
      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
      X-Amz-Date:
      - 20211108T150915Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?acl=
    method: PUT
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Mon, 08 Nov 2021 15:09:16 GMT
      X-Amz-Id-2:
      - txeca05ff9427d4b848d10c-0061893d9c
      X-Amz-Request-Id:
      - txeca05ff9427d4b848d10c-0061893d9c
    status: 200 OK
    code: 200
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=cffed00265952252d9e2633fe595b19acaa72dfcc9941b361610be737c9020c7
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150915Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txd6622e50b1ae4bc98ce08-0061893d9c</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:16 GMT
      X-Amz-Id-2:
      - txd6622e50b1ae4bc98ce08-0061893d9c
      X-Amz-Request-Id:
      - txd6622e50b1ae4bc98ce08-0061893d9c
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=cffed00265952252d9e2633fe595b19acaa72dfcc9941b361610be737c9020c7
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150915Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx87bc507b9f3b4174a46ac-0061893d9e</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:18 GMT
      X-Amz-Id-2:
      - tx87bc507b9f3b4174a46ac-0061893d9e
      X-Amz-Request-Id:
      - tx87bc507b9f3b4174a46ac-0061893d9e
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=cffed00265952252d9e2633fe595b19acaa72dfcc9941b361610be737c9020c7
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150915Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txea719717d4d143969f2be-0061893da2</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:22 GMT
      X-Amz-Id-2:
      - txea719717d4d143969f2be-0061893da2
      X-Amz-Request-Id:
      - txea719717d4d143969f2be-0061893da2
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=cffed00265952252d9e2633fe595b19acaa72dfcc9941b361610be737c9020c7
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150915Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx022f55611fc54144b26b8-0061893daa</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:30 GMT
      X-Amz-Id-2:
      - tx022f55611fc54144b26b8-0061893daa
      X-Amz-Request-Id:
      - tx022f55611fc54144b26b8-0061893daa
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=e420e86e9ff5030b245cfb183887d7fae3807166c1603e4b226727b38bbfaa08
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150929Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx6620a733efac40e0ab8d1-0061893daa</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:30 GMT
      X-Amz-Id-2:
      - tx6620a733efac40e0ab8d1-0061893daa
      X-Amz-Request-Id:
      - tx6620a733efac40e0ab8d1-0061893daa
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=e420e86e9ff5030b245cfb183887d7fae3807166c1603e4b226727b38bbfaa08
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150929Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx56b2a477fbc44956a8919-0061893dac</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:33 GMT
      X-Amz-Id-2:
      - tx56b2a477fbc44956a8919-0061893dac
      X-Amz-Request-Id:
      - tx56b2a477fbc44956a8919-0061893dac
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=e420e86e9ff5030b245cfb183887d7fae3807166c1603e4b226727b38bbfaa08
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150929Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx4095cde5b4164fd38fdcc-0061893db1</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:37 GMT
      X-Amz-Id-2:
      - tx4095cde5b4164fd38fdcc-0061893db1
      X-Amz-Request-Id:
      - tx4095cde5b4164fd38fdcc-0061893db1
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=e420e86e9ff5030b245cfb183887d7fae3807166c1603e4b226727b38bbfaa08
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150929Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx08106ea1f2b54406aa686-0061893db9</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:45 GMT
      X-Amz-Id-2:
      - tx08106ea1f2b54406aa686-0061893db9
      X-Amz-Request-Id:
      - tx08106ea1f2b54406aa686-0061893db9
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6e949e00bd0fea8f567dfce9e86a9a44ce96ce586a7ca2f04b1af868216cf608
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150944Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx93d8d6ede8db42dbbc758-0061893db9</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:45 GMT
      X-Amz-Id-2:
      - tx93d8d6ede8db42dbbc758-0061893db9
      X-Amz-Request-Id:
      - tx93d8d6ede8db42dbbc758-0061893db9
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6e949e00bd0fea8f567dfce9e86a9a44ce96ce586a7ca2f04b1af868216cf608
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150944Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx4e86c000842f4773a2194-0061893dbb</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:47 GMT
      X-Amz-Id-2:
      - tx4e86c000842f4773a2194-0061893dbb
      X-Amz-Request-Id:
      - tx4e86c000842f4773a2194-0061893dbb
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6e949e00bd0fea8f567dfce9e86a9a44ce96ce586a7ca2f04b1af868216cf608
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150944Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txa3433b56918645e89653d-0061893dbf</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:51 GMT
      X-Amz-Id-2:
      - txa3433b56918645e89653d-0061893dbf
      X-Amz-Request-Id:
      - txa3433b56918645e89653d-0061893dbf
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6e949e00bd0fea8f567dfce9e86a9a44ce96ce586a7ca2f04b1af868216cf608
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150944Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx5eff92426f6645dd81727-0061893dc7</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:09:59 GMT
      X-Amz-Id-2:
      - tx5eff92426f6645dd81727-0061893dc7
      X-Amz-Request-Id:
      - tx5eff92426f6645dd81727-0061893dc7
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6a3d05b1ae1982aeaca2dc3a40a162fac8d74cb91b661581907d9598979e51e1
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150958Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx0d051a2eb43844e584899-0061893dc7</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:10:00 GMT
      X-Amz-Id-2:
      - tx0d051a2eb43844e584899-0061893dc7
      X-Amz-Request-Id:
      - tx0d051a2eb43844e584899-0061893dc7
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6a3d05b1ae1982aeaca2dc3a40a162fac8d74cb91b661581907d9598979e51e1
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150958Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx5d82483455e84a19bc919-0061893dca</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:10:02 GMT
      X-Amz-Id-2:
      - tx5d82483455e84a19bc919-0061893dca
      X-Amz-Request-Id:
      - tx5d82483455e84a19bc919-0061893dca
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6a3d05b1ae1982aeaca2dc3a40a162fac8d74cb91b661581907d9598979e51e1
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150958Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx289b29a17d084ce4b6750-0061893dce</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:10:06 GMT
      X-Amz-Id-2:
      - tx289b29a17d084ce4b6750-0061893dce
      X-Amz-Request-Id:
      - tx289b29a17d084ce4b6750-0061893dce
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedHeader>*</AllowedHeader><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6a3d05b1ae1982aeaca2dc3a40a162fac8d74cb91b661581907d9598979e51e1
      Content-Length:
      - "365"
      Content-Md5:
      - doCaNFYhpSrVvrLN1cQaiQ==
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - 62ab1c45dea8e5ff7f728bddc81ca0c025fba503d0afd2a0f282657162c2dcbe
      X-Amz-Date:
      - 20211108T150958Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx20bf18396cd34c42925f7-0061893dd6</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Mon, 08 Nov 2021 15:10:14 GMT
      X-Amz-Id-2:
      - tx20bf18396cd34c42925f7-0061893dd6
      X-Amz-Request-Id:
      - tx20bf18396cd34c42925f7-0061893dd6
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: ""
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCWF4BF54P0CRPCKJRZN/20211108/fr-par/s3/aws4_request,
        SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=37bd99bf4afd1e5a1eef3b4fb51b2a893051408e738c9de6c1ca4788ac282304
      User-Agent:
      - aws-sdk-go/1.41.16 (go1.17.1; darwin; amd64)
      X-Amz-Content-Sha256:
      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
      X-Amz-Date:
      - 20211108T151013Z
    url: https://test-acc-scaleway-object-cors-empty-origin-8035963149870005357.s3.fr-par.scw.cloud/
    method: DELETE
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Mon, 08 Nov 2021 15:10:15 GMT
      X-Amz-Id-2:
      - tx02d0eebb18fe498d846f5-0061893dd6
      X-Amz-Request-Id:
      - tx02d0eebb18fe498d846f5-0061893dd6
    status: 204 No Content
    code: 204
    duration: ""
